hubmap/frontend/cypress/integration/live_auth_test.js

```diff
@@ -0,0 +1,44 @@
+//https://docs.cypress.io/guides/references/error-messages.html#Cypress-detected-a-cross-origin-error-happened-on-page-load
```

Comment:
What is this link for? What question does it answer? (Clarify the code: Response here not needed.)